### PR TITLE
Fixed Growth to be KB when not percent instead of Pages. Also fixed a…

### LINKED ISSUE #2471: Get-DbaDatabaseFile returning wrong 'Growth'
Hello,

Using the command 'Get-DbaDatabaseFile' doesn't return the correct 'Growth'. Screenshots below.

### General Troubleshooting steps
- [x ] Verified running latest release of dbatools?
- [x ] Verified errors are not related to permissions?
- [x ] Can duplicate in new/fresh PowerShell.exe session?

### Version Information
 - Operating System (Name|Version): Windows 10 
 - PowerShell Version: 5.1
 - SQL Server (Edition|Version): Microsoft SQL Server 2017 (RTM) - 14.0.1000.169 (X64)

### Additional information
sql_growth_1

sql_growth_2

### PR DESCRIPTION
**Pull request by dbaduck**
… Select --> Select-Object

## Type of Change

 - [X] Bug fix (non-breaking change, fixes #2471 )
 - [ ] New feature (non-breaking change, adds functionality)
 - [ ] Breaking change (effects multiple commands or functionality)
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 

### Purpose
 
To fix the Growth to not be pages which is stored in the Database but rather KB as it should be.

### Commands to test

Get-DbaDatabaseFile

### CODE REVIEW

**Comment by dbaduck** [comment]
Is there anything else I need to do to get this merged?

**Comment by potatoqualitee** [comment]
Thank you @dbaduck for the improvement! And thanks for the approval, @PowerDBAKlaas! 

My apologies for the delay Ben, I'm overwhelmed right now and depending on others for approvals that require additional thought/tests.

**Comment by wsmelton** [comment]
I can debug this today (I think) and figure out what is going on.

If they run fine in my local environment will need someone to point me on how to RDP into the Appveyor server if we really want to know why it is having the problem.

File: functions/Get-DbaDatabaseFile.ps1 [hunk 1]
```diff
@@ -82,7 +82,7 @@ function Get-DbaDatabaseFile {
             mf.physical_name as PhysicalName,
             df.state_desc as State,
             df.max_size as MaxSize,
-            df.growth as Growth,
+            case mf.is_percent_growth when 1 then df.growth else df.Growth*8 end as Growth,,
```
Comment (wsmelton) [inline functions/Get-DbaDatabaseFile.ps1]: Check the end of the line...that is why the test fails.

AS well around line 172 we need to add a debug statement to output that query. Something like this is what we have been using:

`Write-Message -Level Debug -Message "SQL Statement: $query"`

**Comment by dbaduck** [comment]
Thanks. I am not sure how that extra comma got in. I swear I tested it locally but apparently not the same version as the PR. Thanks for getting this back. I will fix it and I guess I resubmit the PR?

**Pull request merged**